tools/code_format/glint.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
Can you add a human readable comment on the top of each function just to make it clear what is being checked for? Also, do you plan on having an auto fix version? I think this will probably annoy people if we don't auto fix at least the whitespace issues?

Reply:
the problem is the tabs - theres not an easy way of knowing what it should be - editors know

for the other whitespace issues - yep thats easy - can we do in a follow up ?

Reply:
Sure, can you add the comments to the formatter so it's clear what it's doing when people get an error and look at the file?

Reply:
ticket added re autofix - and comments added here